modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -25,6 +25,8 @@ public partial class GodotSharpEditor : EditorPlugin, ISerializationListener
         public static class Settings
         {
             public const string ExternalEditor = "dotnet/editor/external_editor";
+            public const string ExecPath = "dotnet/editor/exec_path";
+            public const string ExecArgs = "dotnet/editor/exec_args"; 
```

Comment:
This should be called "custom_exec_path/args" to avoid confusion.

Reply:
Do you mean 

CustomExecPath = "dotnet/editor/custom_exec_path"
CustomExecPathArgs = "dotnet/editor/custom_exec_path_args" ?

Reply:
Yes, specially the string, as it will be displayed to the user. With "Exec Path" alone, it's not obvious that it goes along with the "Custom" mode.